docs/framework/guides/contributing/code-style.md

```diff
@@ -777,3 +777,126 @@ Widely used standard files do not obey the above rules:
 * `README.md`, `LICENSE.md`, `CONTRIBUTING.md`, `CHANGES.md`
 * `.gitignore` and all standard "dot-files"
 * `node_modules`
+
+## CKEditor 5 custom ESLint rules
+
+In addition to the rules provided by ESLint, CKEditor 5 uses a few custom rules described below.
+
+### Importing between packages: `ckeditor5-rules/no-relative-imports`
+
+While importing modules from the same package, it is allowed to use relative paths, like this:
+
+```js
+// Assume we edit a file located in the path: `packages/ckeditor5-engine/src/model/model.js`
+
+import Position from './position';
+import insertContent from './utils/insertcontent';
+```
+
+While importing modules from other packages, it is not allowed to use relative paths, and the import must be done using the package name, like this:
+
+👎&nbsp; Examples of incorrect code for this rule:
```

Comment:
One thing to consider. There is a description of the rule followed by the "like this", which is followed by the incorrect sample code. This is confusing. If the first code sample is for the incorrect case, the rule description should be followed by "not like this". But IMHO it would be better to not use the "like this" or "not like this" sentences at all.

Reply:
Having thought it over, I would, too, concentrate more on showing examples of good code, while removing the wrong ones. These will google and index, too.

Reply:
When we were writing these sections, we based on https://eslint.org/docs/rules/no-undef.

Reply:
WDYT about removing ", like this" from the sentence?

> While importing modules from other packages, it is not allowed to use relative paths, and the import must be done using the package name.
>
> Example of incorrect code...
> Example of correct code...

Reply:
And this is cool, be we do not really adopt the idea of a good example / bad example in our own docs.

Reply:
I'm all for removing "like this" (even though I live this phrase and have a personal story about it).

Reply:
@Reinmar, WDYT?

Reply:
> And this is cool, be we do not really adopt the idea of a good example / bad example in our own docs.

Why? What's wrong with it? As @pomek mentioned, it's absolutely common in ESLint's documentation which, at least I, consider very well thought and helpful.

Reply:
We just never did it this way in the docs; I will not oppose using this, tho.

Reply:
I can see cases where using bad examples may be confusing (i.e. where they are not clearly marked). But if we'll make sure here that bad examples are clearly marked (in fact, @psmyrek caught an issue with that), doing this, even when for the first time, is IMO fine. We do many things for the first time in this project :D